src/strategies/passport/passport-client-password/client-password-strategy.ts

```diff
@@ -0,0 +1,79 @@
+// Type definitions for passport-oauth2-client-password 0.1.2
+// Project: https://github.com/jaredhanson/passport-oauth2-client-password
+// Definitions by: Ivan Zubok <https://github.com/akaNightmare>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import * as passport from 'passport';
```

Comment:
Why this strategy file added ? Wasn't this already available in passport?

Reply:
there is but for the passport strategy client_secret is mandatory 
so we needed to create out own